bgapi/module.py

```diff
@@ -790,19 +796,23 @@ def ble_rsp_attributes_write(self, result):
         super(BlueGigaServer, self).ble_rsp_attributes_write(result)
         self.procedure_complete(PROCEDURE)
 
+    def ble_rsp_attributes_send(self, result):
+        super(BlueGigaServer, self).ble_rsp_attributes_send(result)
+        self.procedure_complete(SEND_ATTRIBUTE)
+
     def ble_evt_attributes_value(self, connection, reason, handle, offset, value):
         super(BlueGigaServer, self).ble_evt_attributes_value(connection, reason, handle, offset, value)
         self.update_attribute_cache(handle, offset, value)
 
     def ble_rsp_attributes_read_type(self, handle, result, value):
         super(BlueGigaServer, self).ble_rsp_attributes_read_type(handle, result, value)
         self.handle_types[handle] = value
-        self.procedure_complete(PROCEDURE)
+        self.procedure_complete(READ_ATTRIBUTE)
```

Comment:
I feel like READ_ATTRIBUTE is likely a better procedure label for read_type, but I can change it. I did have time-out when it was PROCEDURE, as there was a clash of trying to start a read_type from within the callback of an attributes_value event...